exec/java-exec/src/main/resources/rest/profile/profile.ftl

```diff
@@ -587,6 +622,49 @@
       if (e.target.form) 
         <#if model.isOnlyImpersonationEnabled()>doSubmitQueryWithUserName()<#else>doSubmitQueryWithAutoLimit()</#if>;
     });
+
+    // Convert scientific to Decimal [Ref: https://gist.github.com/jiggzson/b5f489af9ad931e3d186]
+    function scientificToDecimal(num) {
```

Comment:
Why do you need this at all? Javascript parseInt() function which you use can parse scientific format without a problem.

Reply:
I don't believe that works. The parseInt function will parse until the 'E' symbol.

Reply:
From the Chrome web console:
`> parseInt(1.6E7)`
`< 16000000`
So it works it seems.

Reply:
Yep. Verified it. Thanks for pointing it out.